classroom/snippets/src/main/java/CancelGuardianInvitation.java

```diff
@@ -0,0 +1,93 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START classroom_cancel_guardian_invitation_class]
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonError;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.gson.GsonFactory;
+import com.google.api.services.classroom.Classroom;
+import com.google.api.services.classroom.ClassroomScopes;
+import com.google.api.services.classroom.model.GuardianInvitation;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+
+/* Class to demonstrate the use of Classroom Patch Guardian Invitation API. */
+public class CancelGuardianInvitation {
+  /**
+   * Cancel a guardian invitation by modifying the state of the invite.
+   *
+   * @param studentId - the id of the student.
+   * @param invitationId - the id of the guardian invitation to modify.
+   * @return - the modified guardian invitation.
+   * @throws IOException - if credentials file not found.
+   */
+  public static GuardianInvitation cancelGuardianInvitation(String studentId, String invitationId)
+      throws Exception {
+
+    /* Scopes required by this API call. If modifying these scopes, delete your previously saved
+    tokens/ folder. */
+    final List<String> SCOPES =
+        Collections.singletonList(ClassroomScopes.CLASSROOM_GUARDIANLINKS_STUDENTS);
+
+    // Create the classroom API client
+    ClassroomCredentials classroomCredentials = new ClassroomCredentials();
+    final NetHttpTransport HTTP_TRANSPORT = GoogleNetHttpTransport.newTrustedTransport();
+    Classroom service = new Classroom.Builder(HTTP_TRANSPORT,
+        GsonFactory.getDefaultInstance(),
+        classroomCredentials.getCredentials(HTTP_TRANSPORT, SCOPES))
```

Comment:
It looks like everything in the `ClassroomCredentials` class is `static`. Why do we need a constructed instance of it? Could we not just call `ClassroomCredentials.getCredentials(HTTP_TRANSPORT, SCOPES)` directly?

Reply:
You're right. Updated.